From: Serge Hallyn <serge.hallyn@canonical.com>
Date: Wed, 30 Jan 2013 21:05:45 -0600
Subject: complete virterror->virerror name change

Without these two string changes in generator.py, the
virGetLastError wrapper does not get created in
/usr/share/pyshared/libvirt.py.  Noticed when running
tests with virt-install.

Signed-off-by: Serge Hallyn <serge.hallyn@ubuntu.com>
---
 python/generator.py |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/generator.py b/python/generator.py
index 9297a15..4e4642f 100755
--- a/python/generator.py
+++ b/python/generator.py
@@ -124,7 +124,7 @@ class docParser(xml.sax.handler.ContentHandler):
         elif tag == 'enum':
             # enums come from header files, hence virterror.h
             if (attrs['file'] == "libvirt" or
-                attrs['file'] == "virterror"):
+                attrs['file'] == "virerror"):
                 enum(attrs['type'],attrs['name'],attrs['value'])
             elif attrs['file'] == "libvirt-lxc":
                 lxc_enum(attrs['type'],attrs['name'],attrs['value'])
@@ -139,7 +139,7 @@ class docParser(xml.sax.handler.ContentHandler):
             if self.function != None:
                 if (self.function_module == "libvirt" or
                     self.function_module == "virevent" or
-                    self.function_module == "virterror"):
+                    self.function_module == "virerror"):
                     function(self.function, self.function_descr,
                              self.function_return, self.function_args,
                              self.function_file, self.function_module,
